redcap_bridge/test_redcap/test_server_elab_interface.py

```diff
@@ -0,0 +1,41 @@
+from redcap_bridge.server_elab_interface import (create_template_without_metadata, create_template_with_metadata,
+                                                 create_template_with_converted_csv, download_experiment)
+from redcap_bridge.test_redcap.test_utils import (test_directory, initialize_test_dir)
+
+SERVER_CONFIG_YAML = (test_directory / 'testfiles' / 'TestProject' / 'project.json').resolve()
+
+test_file = (test_directory / 'testfiles' / 'elabConversion' / 'csvRecord.csv')
+
+
+def test_create_template_without_metadata(initialize_test_dir):
+    template_file = test_directory / 'testfiles' / 'elab_template.json'
+
+    res = create_template_without_metadata(server_config_json=SERVER_CONFIG_YAML, template_file=template_file)
+
+    assert res is not None
+
+
+def test_create_template_with_metadata(initialize_test_dir):
+    template_file = test_directory / 'testfiles' / 'elab_template.json'
+
+    res = create_template_with_metadata(server_config_json=SERVER_CONFIG_YAML, template_file=template_file)
+
+    assert res is not None
+
+
+def test_create_template_with_converted_csv(initialize_test_dir):
+    csv_file = test_directory / 'testfiles' / 'metadata.csv'
+
+    res = create_template_with_converted_csv(server_config_json=SERVER_CONFIG_YAML, csv_file=csv_file,
+                                             title='General Template')
+
+    assert res is not None
+
+
+def test_download_experiment(initialize_test_dir):
+
+    res = download_experiment(server_config_json=SERVER_CONFIG_YAML, id=232)
+
+    print(res)
```

Comment:
Tests should not use print statements as these will clutter the output when large test suits are run.
```suggestion
```